src/Components/LinkCreator/LinkCreator.jsx

```diff
@@ -1,16 +1,20 @@
-import React, { useState, useEffect } from "react";
```

Comment:
keep `React`

Reply:
But React is unused and eslint would mark it. I would have to disable this line